controllers/clusterenvironment_controller.go

```diff
@@ -176,6 +177,49 @@ func (r *ClusterEnvironmentReconciler) Reconcile(ctx context.Context, req ctrl.R
 	return ctrl.Result{}, nil
 }
 
+func (r *ClusterEnvironmentReconciler) MonitorHealth(ctx context.Context, ns string, hci int) {
```

Comment:
I'd propose to move this in `pkg/primaza/healthchecks/clusterenvironment_healthchecker.go`